src/integrationtests/java/com/aws/greengrass/integrationtests/logmanager/LogManagerTest.java

```diff
@@ -177,22 +181,19 @@ void GIVEN_user_component_config_with_small_periodic_interval_WHEN_interval_elap
         assertEquals(1, putLogEventsRequests.size());
         for (PutLogEventsRequest request : putLogEventsRequests) {
             assertEquals(calculateLogStreamName(THING_NAME, LOCAL_DEPLOYMENT_GROUP_NAME), request.logStreamName());
-            assertEquals("/aws/greengrass/UserComponent/" + AWS_REGION + "/UserComponentA",
-                    request.logGroupName());
+            assertEquals("/aws/greengrass/UserComponent/" + AWS_REGION + "/UserComponentA", request.logGroupName());
             assertNotNull(request.logEvents());
-            assertEquals(50, request.logEvents().size());
-            assertEquals(51200, request.logEvents().stream().mapToLong(value -> value.message().length())
-                    .sum());
+            assertEquals(DEFAULT_LOG_LINE_IN_FILE * 6, request.logEvents().size());
```

Comment:
It is not completely obvious what 6 means here. Is there a way it could be more descriptive? - like 

```java
// At the top of the test
LogFileGroup group = LogFileGroup.create(
   Pattern.compile("^integTestRandomLogFiles.log\\w*"), tempDirectoryPath.toUri(), Instant.now())

// Create the log file etc etc

// Inside the for loop
assertEquals(DEFAULT_LOG_LINE_IN_FILE * group.size(), request.logEvents().size());
```

Also this will remove the need for that weird for loop we have everywhere